src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlValueConverter.cs

```diff
@@ -272,22 +278,29 @@ protected XmlBaseConverter(XmlBaseConverter converterAtomic)
         {
             _schemaType = converterAtomic._schemaType;
             _typeCode = converterAtomic._typeCode;
-            _clrTypeDefault = Array.CreateInstance(converterAtomic.DefaultClrType!, 0).GetType();
+            _clrTypeDefault = converterAtomic.DefaultClrListType;
+            Debug.Assert(_clrTypeDefault!.IsArray);
+            // We don't support lists-of-lists, so we don't set _clrListTypeDefault to null.
+            _clrListTypeDefault = null;
```

Comment:
```suggestion
            // We don't support lists-of-lists, so we set _clrListTypeDefault to null.
            _clrListTypeDefault = null;
```

Is `don't` extraneous here?